packages/sirius-web/frontend/sirius-web-application/src/footer/Footer.types.ts

```diff
@@ -0,0 +1,14 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+export interface FooterProps {}
```

Comment:
Is this interface really needed?
I think it can be removed

Reply:
I would agree in general but we want to "type" the kind of component expected so it's a bit necessary even to indicate that there won't be any props.